compiler/src/main/scala/org/scalajs/core/compiler/PrepJSInterop.scala

```diff
@@ -1027,6 +1031,17 @@ abstract class PrepJSInterop extends plugins.PluginComponent
       ddef
     }
   }
+
+  private def checkInternalAnnotations(sym: Symbol): Unit = {
+    for {
+      ann <- compilerInternalAnnotations
+      annSym <- sym.getAnnotation(ann)
+      if annSym.pos.isDefined
```

Comment:
Ok, that would eliminate most of them. There would remain the case where the user writes it fully qualified, any ideas for that one?